pkgs/top-level/aliases.nix

```diff
@@ -367,9 +367,11 @@ mapAliases ({
   gnupg20 = throw "gnupg20 has been removed from nixpkgs as upstream dropped support on 2017-12-31";# added 2020-07-12
   gnuvd = throw "gnuvd was removed because the backend service is missing"; # added 2020-01-14
   gmock = gtest; # moved from top-level 2021-03-14
+  go_1_12 = throw "go_1_12 has been removed"; # added 2020-04-26
```

Comment:
Assuming this was readded by accident after it was removed in https://github.com/NixOS/nixpkgs/pull/157334.

I've  removed again in https://github.com/NixOS/nixpkgs/pull/158885.

Reply:
Ah, yes. Probably screwed that up during cherry-pick and rebase, somehow.
Sorry for not checking again properly.